angular/src/components/settings/vault-timeout-input.component.ts

```diff
@@ -132,6 +134,10 @@ export class VaultTimeoutInputComponent implements ControlValueAccessor, Validat
       return { policyError: true };
     }
 
+    //Check minimum allowed custom minutes
+    this.validMinutes =
+      control.value < VaultTimeoutInputComponent.MIN_CUSTOM_MINUTES ? false : true;
```

Comment:
We don't need to assign this to a variable, simply return an error and handle the error in the form validation logic in the clients.